pennylane_lightning/src/simulator/Measures.hpp

```diff
@@ -42,12 +42,12 @@ namespace Pennylane {
 template <class fp_t = double, class SVType = StateVectorRaw<fp_t>>
 class Measures {
   private:
-    const SVType &original_statevector;
+    const SVType original_statevector;
     using CFP_t = std::complex<fp_t>;
 
   public:
-    Measures(const SVType &provided_statevector)
-        : original_statevector{provided_statevector} {};
+    Measures(const SVType provided_statevector)
```

Comment:
Can a const copied element be moved? If so, I am surprised the standard allows it.

Reply:
It silently ignores `move`. https://www.youtube.com/watch?v=ZKaoR3dP9uM

Reply:
Huh, I had no idea.